zerver/views/thumbnail.py

```diff
@@ -27,12 +27,13 @@ def validate_thumbnail_request(
     return True
 
 
-@has_request_variables
+@typed_endpoint
 def backend_serve_thumbnail(
     request: HttpRequest,
     maybe_user_profile: Union[UserProfile, AnonymousUser],
-    url: str = REQ(),
-    size_requested: str = REQ("size"),
+    *,
+    url: str,
+    size: str,
```

Comment:
So this is a functional change, but I guess `size_requested` was unused pending this feature being re-added, so probably it's fine. Please try to avoid changing things without comment like this in a migration PR, it makes it harder to verify your work.

Reply:
Ok, I'll be a little more careful with that in the future. Since it was unused, I tried reducing the imports, therefore I  didn't want to import `ApiParamConfig`.